src/current/v24.1/cockroach-start.md

```diff
@@ -258,24 +258,24 @@ To enable WAL failover, you must take one of the following actions:
-- If you must log to local disks:
-  1. Disable [audit logging]({% link {{ page.version.version }}/sql-audit-logging.md %}). File-based audit logging and the WAL failover feature cannot coexist. File-based audit logging provides guarantees that every log message makes it to disk, otherwise CockroachDB needs to shut down. Because of this, resuming operations in the face of disk unavailability is not compatible with audit logging.
-  1. Enable asynchronous buffering of [`file-groups` log sinks]({% link {{ page.version.version }}/configure-logs.md %}#output-to-files) using the `buffering` configuration option. The `buffering` configuration can be applied to [`file-defaults`]({% link {{ page.version.version }}/configure-logs.md %}#configure-logging-defaults) or individual `file-groups` as needed. Note that enabling asynchronous buffering of `file-groups` log sinks is in [preview]({% link {{ page.version.version }}/cockroachdb-feature-availability.md %}#features-in-preview).
-  1. Set `max-staleness: 1s` and `flush-trigger-size: 256KiB`.
-  1. When `buffering` is enabled, `buffered-writes` must be explicitly disabled as shown below. This is necessary because `buffered-writes` does not provide true asynchronous disk access, but rather a small buffer. If the small buffer fills up, it can cause internal routines performing logging operations to hang. This in turn will cause internal routines doing other important work to hang, potentially affecting cluster stability.
-  1. The recommended logging configuration for using file-based logging with WAL failover is as follows:
-
-        ~~~
-        file-defaults:
-          buffered-writes: false
-          buffering:
-            max-staleness: 1s
-            flush-trigger-size: 256KiB
-            max-buffer-size: 50MiB
-        ~~~
+Therefore, if you enable WAL failover and log to local disks, you must also update your [logging]({% link {{page.version.version}}/logging-overview.md %}) configuration as follows:
+
+1. Disable [audit logging]({% link {{ page.version.version }}/sql-audit-logging.md %}). File-based audit logging and the WAL failover feature cannot coexist. File-based audit logging provides guarantees that every log message makes it to disk, otherwise CockroachDB needs to shut down. Because of this, resuming operations in the face of disk unavailability is not compatible with audit logging.
+1. Enable asynchronous buffering of [`file-groups` log sinks]({% link {{ page.version.version }}/configure-logs.md %}#output-to-files) using the `buffering` configuration option. The `buffering` configuration can be applied to [`file-defaults`]({% link {{ page.version.version }}/configure-logs.md %}#configure-logging-defaults) or individual `file-groups` as needed. Note that enabling asynchronous buffering of `file-groups` log sinks is in [preview]({% link {{page.version.version}}/cockroachdb-feature-availability.md %}#features-in-preview).
```

Comment:
This confused me because the `buffering` option isn't documented in the table at `configure-logs.html#output-to-files`. Do you know if we have a separate ticket to cover that? It's also not listed in the linked 'preview features' doc as far as I can tell.

Reply:
ya it's a bit confusing, since this feature is not GA we didn't yet doc the log buffering stuff

from my other comment:

> totes agreed, right now it's experimental per https://cockroachlabs.atlassian.net/browse/DOC-9925 bcs it was paired with the WAL failover stuff

> but we have https://cockroachlabs.atlassian.net/browse/DOC-10240 which will add it when it becomes GA